docs/architecture.md

```diff
@@ -0,0 +1,47 @@
+# Architecture
+
+What makes up the different actors and components of the DoubleZero network?
+
+Figure 1: Network architecture components
+
+## Contributors
+
+The DoubleZero network is made up of contributions from a growing community of distributed systems operators and network infrastructure providers in cities across the globe. Contributors bring physical link and/or compute resources to the network to provide the decentralized mesh network
+
+### Bandwidth
+
+The physical links of the DoubleZero network, most commonly in the form of fiberoptic cables or commonly referred to as wavelength services. Bandwidth contributors commit underutilized network links, owned or leased from infrastructure providers, between two or more data centers. Bandwidth links are terminated at both ends by DoubleZero Devices, physical network switching enclosures running instances of the DoubleZero Agent software.
+
+#### DoubleZero Exchange (DZX / Cross-connect Site)
+
+DoubleZero Exchanges (DZXs) are interconnect points in the mesh network where different contributor links are bridged together. DZXs are located in major metropolitan areas around the world where network intersections occur. Network bandwidth contributors pay to cross-connect their links into the broader DoubleZero mesh network at the DZXs geographically located nearest their link endpoints.
+
+### Resource
+
+Resource contributors host instances of the DoubleZero Ledger, providing the RPC endpoints to the rest of the DoubleZero network for users to configure the network and devices to read that configuration.
+
+## Nodes
```

Comment:
Would it make sense to call this section `Components` instead? Depends on how you're defining the word node but ledger is more of a blockchain/network than a single node, and agent is more of an embedded process on devices.